content/docs/user-guide/settings/import-settings-registry.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Settings Registry $import Chaining
+linkTitle: Settings Registry $import
+description: Learn how the Settings Registry can use the $import directive to merge additional settings registry files in Open 3D Engine (O3DE).
+weight: 300
+---
+
+The Settings Registry supports the ability to merge additional Settings Registry files `*.setreg(patch)` via the [JSON Importer Framework](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Serialization/Json/JsonImporter.h).  
+Any field named `$import` within a Settings Registry file cab be used to specify additional `*.setreg`, `*.setregpatch` or any other JSON format files to merge to the Settings Registry.  
+
+When a Settings Registry file is merged using the [Merge API](./developer-api#merge-api), it triggers the logic to look for `$import` directives within the settings file to merge additional JSON files.
+The order in which the additional files are merged occur at the point where their `$import` directives are processed.  
+Any setting that appears before the `$import` directive can be overridden by the JSON contents of the imported file.  
+Any setting that appears after the `$import` directive will override and merge over the JSON contents of the imported file.
+
+# Chain import setreg file using a `$import` directive string
+
+The following shows how to use the `$import` directive to merge another Settings Registry file using a string value.
+
+### test.apple.setreg
+```json
+{
+    "pre_field": { "first": 1, "second": 2 },
+    "$import": "test.ios.setreg",
+    "post_field": { "1": 11, "2": 12 }
+}
+```
+
+### test.ios.setreg
+```json
+{
+    "pre_field": { "second": 202 },
+    "post_field": { "2": 120 }
+}
+```
+
+The merging the `test.apple.setreg` file results in the final JSON output that follows.
+
+### Import Result
+```json
+{
+    "pre_field": { "first": 1, "second": 202 },
+    "post_field": { "1": 11, "2": 12 }
+}
+```
+
+Notice that the `"pre_field"` that appear before the `$import` directive has its `"second"` field value (`202`) overridden by the imported `test.ios.setreg` file.  
+Contrast the difference with the `"post_field"` that appears after the `$import` directive. The `"2"` field value (`12`) is from the original `test.apple.setreg` file.
+
+# Chain import setreg file using a `$import` directive object
+
+The following shows how to use the `$import` directive to merge another Settings Registry file using a JSON object with a `filename` key.
```

Comment:
Can you give a reason / scenario why a user would use this more complex syntax?

Reply:
Yes, this syntax comes into play, if for example the user wanted to apply a JSON merge patch operation of a specific JSON string directly over the imported contents.

The `$import` directive supports a `patch` field, which has been undocumented so far, but is being used with the JSON Serialization system as part of the Material Type JSON files [here](https://github.com/o3de/o3de/blob/2581515db411ec90f633a2414d174fab638da559/Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.materialtype#L330-L338).

I'll add a mention of the `patch` field here and will properly document it, so that the user has a reason to use the object syntax.